float32 SpeedOfSoundInWaterMetersPerSecond      # this is a value from 1400.0 m/s to 1600.0 m/s. The default is 1500.00 m/s
